packages/tokens/bin/lib/formats/formatSvg.ts

```diff
@@ -0,0 +1,51 @@
+import {camelCase} from 'lodash';
+import {optimize, OptimizedSvg} from 'svgo';
+
+import {isTokenEnum, isTokenGroup, Token, TokenEnum, TokenGroup, TokenList} from './token';
+
+export type SvgFormatOutput = {fileName: string; svgMarkup: string};
+
+const cleanSvgMarkup = (name: string, svgMarkup: string): string => {
+    const result = optimize(svgMarkup, {
+        multipass: true,
+        plugins: [
+            {name: 'removeViewBox', active: false},
+            {name: 'removeDimensions', active: true},
+            {
+                name: 'removeAttributesBySelector',
+                active: true,
+                params: {selector: '[stroke]', attributes: ['stroke']} as any,
+            },
+            {
+                name: 'addAttributesToSVGElement',
+                active: true,
+                params: {attribute: {stroke: 'currentColor'}} as any,
```

Comment:
So it gives us this in the svg files, is it normal? Shouldn't we have a color in Hexa?
`stroke="currentColor"`

Reply:
https://css-tricks.com/currentcolor/